llvm/lib/Target/NVPTX/NVPTXInstrInfo.td

```diff
@@ -876,85 +874,6 @@ let Predicates = [doMulWide] in {
   def : Pat<(i64 (mul_wide_unsigned i32:$a, imm:$b)), (MULWIDEU64Imm $a, imm:$b)>;
 }
 
-// Predicates used for converting some patterns to mul.wide.
```

Comment:
Why are we able to just remove this whole section? Are these transformations already handled somewhere else?

Reply:
Yes, I believe they're already handled by `TryMULWIDECombine`.